prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -194,6 +195,21 @@ function download_exporter () {
     # cleanup what was downloaded
     rm -rf ${EXPORTER_BASE_NAME}*
   fi
+
+  if [ "$EXPORTER" == "postgres" ]; then
+    EXPORTER_VERSION="0.10.1"
+    EXPORTER_BASE_NAME="postgres_exporter-${EXPORTER_VERSION}.aix-ppc64"
```

Comment:
remove this , I think machine Aix is not what we are using

Reply:
Removed this